### ISSUE TITLE
How to select features when needing to perform extra computation on the selected features before feeding into SVM?

### ISSUE THREAD

**Issue by imxtx** [rodrigo-arenas/Sklearn-genetic-opt#139]
The **Feature Selection** example is good, but how to select features when needing to perform extra computation on the selected features before feeding into SVM? 

Here is an example: `X` are features with a shape `[num_example, 1000]` and `y` are labels with a shape `[num_example,1]`. I want to select 100 features from 1000 features. The selected features can form a new training data `X'` with a shape `[num_example, 100]`. I want an SVM model to fit on `cos(X'[num_example, 0:50], X'[num_example, 50:100], axis=1)` and labels `y`. 

How should I do this?

**Comment by rodrigo-arenas** [comment]
Hi @imxtx, if I understand correctly, you want to use only the subset of selected features, you can achieve this with different approaches

One is to directly call the predict method or your evolved_estimator, to which you already passed the SVM, so you don't need to fit it again

```python
evolved_estimator.fit(X, y)

# Predict only with the subset of selected features
y_predict_ga = evolved_estimator.predict(X_test)

```

The second option, if you want to create another model using the selected features, is to use the transform method, which will take the X and return only X' with the subset of selected features

```python
new_X = evolved_estimator.transform(X_test)

```

You can also inspect which features were selected and which ones weren't with this method

```python
evolved_estimator.support_

```

I hope it helps

**Comment by imxtx** [comment]
Thank you @rodrigo-arenas ! BTW, I created pull request, can you check it?
